src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+        >
+
+      <fieldset>
+        <legend>Personal information</legend>
+        Surname
+        <input
+          type="text"
+          name="surname"
+          required
+          autocomplete="off"><br>
+
+        Name:
+        <input
+          type="text"
+          name="name"
+          required
+          autocomplete="off"><br>
```

Comment:
```suggestion
        <input
          type="text"
          name="name"
          required
          autocomplete="off"
        >
```
You don't need tag <br>